src/App.js

```diff
@@ -1,8 +1,37 @@
 import './App.css';
+import Tasks from './todo.json'
```

Comment:
@PuniaMohit todo.json exporting Tasks? why this much issue in naming conventions?